src/extension/ext-raw.js

```diff
@@ -101,6 +107,19 @@ export default {
               backgroundColor: {
                 type: 'items',
                 items: {
+                  navigation: {
+                    type: 'items',
+                    items: {
+                      navigtaionMode: {
+                        ref: 'navigationMode',
+                        type: 'string',
+                        translation: '$Navigationmode',
+                        component: 'dropdown',
+                        default: 'scroll',
+                        options: navigationOptions,
+                      },
+                    },
+                  },
```

Comment:
Add this stuff in, but for now set default to free and hide the option. That way we can publish the chart without exposing the other options for now.

Reply:
We want to allow for automatic as well right? Should be able to set that in property panel I guess